lms/djangoapps/support/static/support/jsx/logged_in_user.jsx

```diff
@@ -131,13 +131,13 @@ function LoggedInUser({
             <div className="row">
                 <div
                     className="col-sm-12 user-info"
-                    data-username={userInformation.username}
+                    data-name={userInformation.full_name}
```

Comment:
keep username and add name
```suggestion
                    data-username={userInformation.username} 
                    data-name={userInformation.full_name}
```